community/vncsnapshot/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Ishak BELAHMAR <ishak@nytrio.com>
+# Maintainer: Ishak BELAHMAR <ishak@nytrio.com>
+pkgname=vncsnapshot
+pkgver=1.3
+pkgrel=0
+pkgdesc="The utility allows one to take a snapshot from a VNC server and save it as a JPEG file."
+url="https://github.com/IDNT/vncsnapshot"
+arch="x86_64"
+license="GPL-2.0"
+depends=""
+makedepends="jpeg-dev zlib-dev"
+install=""
+subpackages="$pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/IDNT/vncsnapshot/archive/master.tar.gz"
```

Comment:
Not a tagged release. I recommend grabbing the source from https://sourceforge.net/projects/vncsnapshot/files/vncsnapshot/1.2a/vncsnapshot-1.2a-src.tar.gz/download

and then turning https://github.com/IDNT/vncsnapshot/commit/65f9532985d288dc332586e548c7648633ac69e5 into a patch. The version pkgver should then be 1.2a.